tests/spec/core/id-headers-spec.js

```diff
@@ -33,7 +33,7 @@ describe("Core - ID headers", () => {
 
   it("sets an id on header", () => {
     const h2 = doc.querySelector("#t0 h2");
-    expect(h2.id).toBe("foo");
+    expect(h2.id).toBe("x1-foo");
```

Comment:
Oh. This seems to be a breaking change. IIUC, it's supposed to be adding id `"foo"` to `<section>` now (but that section already has an id)?

Reply:
Concerned with any sort of xref/webref breakage mainly.

Reply:
This is consistent with "normal" mode, so it should be fine. Also, both ids (the one on the section and the one on the header are retained, so nothing will break.)